webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrength.js

```diff
@@ -22,9 +23,11 @@ export default class PasswordStrength extends React.Component {
         ? userInputIds.map(input => document.getElementById(input).value)
         : [];
 
+    const showErr = (present, err) => !present && err;
```

Comment:
nitpick: this function is redundant,  on each render it will be recreated.
 it would be better just to store the bool value:
```javascript
  const showErr = !passwordPresent && error
```